aws/alb/versions.tf

```diff
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = ">= 0.12"
```

Comment:
Do you want to specify a version constraint that allows 0.12 versions but stops before 0.13 (since in the 0.x releases there can be backwards breaking changes)?

Reply:
(and similarly elsewhere)

Reply:
The versions.tf files were created by the 'terraform 0.12upgrade' command.  Some of the 0.12 changes are invalid in 0.11.  I chose not to adjust the version constraint.

Reply:
That makes sense. I was just wondering whether the new version constraint was too permissive (since it would presumably allow 0.13 versions). However, if it's what the upgrade process generated, I'm fine with leaving it.